### ISSUE TITLE
How to Ignore Specified Custom Modules

### ISSUE THREAD

**Issue by jenkey2011** [vuejs/eslint-plugin-vue#957]
**Please describe what the rule should do:**

Ignore Specified Custom Modules ro make it possible to skip checks

**What category should the rule belong to?**

- [ ] Enforces code style
- [ ] Warns about a potential error
- [x] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
  <div />
</template>

<script type="customBlock">
// The code here skips checking
// The following code is in a convention format, but it's not legal
{
  "foo": "bar"
}
</script>

<script>
export default {
  // everything ok
}
</script>
```

**Additional context**

**Comment by ota-meshi** [comment]
Hi.

Only one `<script>` can be defined for eslint-plugin-vue to work.
